### PR TITLE
improve tests.pyproj for intellisense and running tests

### PR DESCRIPTION
**Pull request by denfromufa**
### What does this implement/fix? Explain your changes.
adds missing python scripts, adds search path (still need to add ..\..\build\..\ somehow)

### Does this close any currently open issues?
No.

### Any other comments?
NA

### Checklist

Check all those that are applicable and complete.

-   [ ] Make sure to include one or more tests for your change - NA
-   [ ] If an enhancement PR, please create docs and at best an example - NA
-   [X] Add yourself to [`AUTHORS`](../blob/master/AUTHORS.md)
-   [ ] Updated the [`CHANGELOG`](../blob/master/CHANGELOG.md) - NA

### CODE REVIEW

**Comment by denfromufa** [comment]
@vmuriart should it be `from _compat import input`?

File: src/tests/tests.pyproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <ProjectGuid>{250c535c-c060-4f0c-bd80-41f2bf373565}</ProjectGuid>
     <ProjectHome />
     <StartupFile>runtests.py</StartupFile>
-    <SearchPath />
+    <SearchPath>..\..\</SearchPath>
```
Comment (vmuriart) [inline src/tests/tests.pyproj]: whats this do?

**Comment by vmuriart** [comment]
@denfromufa are you refering to the `run_tests.py`? 

A better solution is to change 

```python
if __name__ == '__main__':
    main()
    if '--pause' in sys.argv:
        print("Press enter to continue")
        input()
```

to 

```python
if __name__ == '__main__':
    main()
```

I doubt anyone actually uses the `--pause` flag. I would also add a deprecation comment on the file to ask users to simply run `pytest` instead.

File: src/tests/tests.pyproj (hunk 1 above)
Comment (denfromufa) [inline src/tests/tests.pyproj]: This adds root folder of pythonnet to search paths. Helps when building with --inplace option.

**Comment by denfromufa** [comment]
@vmuriart pause is only helpful when debugging to attach to the python process.

My concern was not with input, but how relative import for _compat module is done.

**Comment by vmuriart** [comment]
give it a try without the period. If i remember right, all other modules will fail without it. This was a while ago though, there might have been other issues that were causing it though

**Comment by denfromufa** [comment]
`from ._compat import input` seems to be fine now.

**Pull request merged**